hackathon_site/dashboard/frontend/src/pages/Orders/Orders.test.tsx

```diff
@@ -21,38 +21,39 @@ describe("Orders Page", () => {
         expect(getByTestId("ordersCountDivider")).toBeInTheDocument();
     });
 
-    test("Display correct order cards", () => {
```

Comment:
Personally I think we shouldn't keep commented code, you can remove them if you would like.